peer_protocol/piece.go

```diff
@@ -0,0 +1,36 @@
+package peer_protocol
```

Comment:
look at https://golang.org/doc/effective_go.html#package-names for a good example for package names